openslides/agenda/projector.py

```diff
@@ -38,6 +38,11 @@ def get_requirements(self, config_entry):
         yield ProjectorRequirement(
             view_class=ItemViewSet,
             view_action='list')
+        for speaker in self.speakers.all():
```

Comment:
this is a bit on confusion on my side. I will need them for the list of speakers overlay, but this belongs in the corresponding PR